pkg/pipeline/transform/transform_network.go

```diff
@@ -182,6 +186,47 @@ func fillInK8s(outputEntry config.GenericMap, rule api.NetworkTransformRule) {
 	}
 }
 
+func fillInK8sInfra(outputEntry config.GenericMap, rule api.NetworkTransformRule) {
+	if rule.KubernetesInfra == nil {
+		logrus.Error("transformation rule: Missing Kubernetes Infra configuration ")
+		return
+	}
```

Comment:
Should we provide a default behavior here ?

Reply:
The case here happens if a user added a K8sinfra rule but did not configure it.

In this case, for now we ignore the rule and log an error.